### DISCUSSION TITLE
Auth / Chat

### DISCUSSION THREAD

**Discussion by BuchananQuantum** [WalletConnect/walletconnect-monorepo#1360] [Developer Support]
I'm sure this is an unknown in terms of exact launch date but does the team have a rough timeframe for the release? 

Will this be out in Q4? 

In terms of budgeting for an upcoming project I'd rather ensure the tech is up to par prior to contracting for development.

**Comment by chadyj** [comment]
We are actively working on Auth and have working internal demos. Expect an alpha in September, and a beta in October.

**Reply by BuchananQuantum** [reply]
Did you release a beta or alpha?

**Reply by chadyj** [reply]
Sure did! Here are some links for Auth.

https://docs.walletconnect.com/2.0/introduction/auth

https://medium.com/walletconnect/introducing-walletconnect-auth-one-click-wallet-login-to-simplify-web3-ux-9cc11fd18aba
